addons/web/static/src/core/emoji_picker/emoji_picker.js

```diff
@@ -506,10 +510,22 @@ export class EmojiPicker extends Component {
         return [...this.recentEmojis, ...this.getEmojis()];
     }
 
+    onKeydownCategory(ev, categoryId) {
+        if (["Enter", "Space"].includes(ev.code)) {
+            this.selectCategory(categoryId);
+        }
+    }
```

Comment:
We could also have arrow left/right for category selection.

Reply:
This doesn't work well since we put the focus back inside the emoji list

Reply:
Ok, good enough ;)